common_rest/src/main/java/uk/gov/dhsc/htbhf/errorhandler/ErrorHandler.java

```diff
@@ -140,4 +143,26 @@ private ErrorResponse convertAndLogBindingResult(BindingResult bindingResult) {
         }
         return super.handleExceptionInternal(ex, response, headers, status, request);
     }
+
+    protected String constructExceptionDetail(Exception exception) {
```

Comment:
Instead of simply logging `exception.getMessage()` we're now consolidating details of the exception, the line it was thrown, and all causes, is a single line of text so it will all appear in one message in logit.io.